README.md

```diff
@@ -1,79 +1,132 @@
 # Istio Apigee Adapter
 
-This workspace holds an Apigee adapter for Istio Mixer. It can be tested standalone as noted below.
-Instructions for building and running in Kubernetes are available here: [README-Kubernetes.MD]().
+This is the source repository for Apigee's Istio Mixer Adapter. This allows users of Istio to
+incorporate Apigee Authentication, Authorization, and Analytics policies to protect and
+report through the Apigee UI.
 
-Note: This repo should be in $GOPATH/src/github.com/apigee/istio-mixer-adapter
+# Installation and usage
 
-Note: You can only build Istio Docker images on Linux.
+## Install Istio with Apigee mixer
 
-## Building and testing standalone
+The quickest way to get started is to follow the [Istio Kubernetes Quick Start](https://istio.io/docs/setup/kubernetes/quick-start.html).
 
-1. Install protoc and dep prerequisites:
+Before you install Istio into Kubernetes (step 5: Install Istio’s core components), edit the `istio.yaml` or `istio-auth.yaml` file to point to the Apigee mixer instead of the Istio vanilla mixer.
 
-        [](https://developers.google.com/protocol-buffers/docs/downloads) 
-        [](https://github.com/golang/dep)
+Assuming Istio 0.6.0:
 
-2. Run install script.
+Replace: `docker.io/istio/mixer:0.6.0` with: `docker.io/robbrit/istio-mixer:nightly`
```

Comment:
gcr.io/apigee-api-management-istio/istio-mixer:nightly, the robbrit one is gone.